package.json

```diff
@@ -25,22 +25,24 @@
   },
   "scripts": {
     "test": "echo \"Error: no test specified, \"$npm_package_version && exit 1",
-    "install": "gulp",
+    "build": "gulp",
```

Comment:
No longer using npm install, build won't result in install being called each time when npm install from a consumer of remote-pay-cloud.